src/docs/product/dashboards/widget-builder/index.mdx

```diff
@@ -0,0 +1,128 @@
+
+### Table
+
+Table charts display the results of a [Discover Query](/product/discover-queries/). This visualization is well suited to showing key fields and related aggregates. For example “duration percentiles for the most frequently visited transaction names”. You can also choose to display [issues](/product/issues/) data in a table visualization using the [dataset selector](/product/dashboards/widget-builder/#choose-your-dataset).
+
+### World Map
+
+Events are queried using the given function and `geo.country_code`. The results are plotted as a density value on a world map. An example scenario would be “in which countries are users experiencing the most errors”. You can zoom the map using the mouse wheel, or pan the map by clicking and dragging the map around.
+
+### Big Number
+
+A big number visualization displays the current value of a single function. This visualization is well suited for high-level aggregates. An example scenario would be “p95 of all transactions”.
+
+## Choose Your Dataset
+
+In the dataset selection step, choose which type of data you would like to use in your widget. This data is classified into two different datasets: [errors and transactions](#errors--transactions) and [issues](#issues).
+
+### Errors & Transactions
+
+Choosing "Errors and Transactions" allows you to query and aggregate error and transaction events in the same way you would for a [Discover Query](/product/discover-queries/). Some widget examples include: 
+
+- Tracking performance of an endpoint 
+- Throughput by country 
+- Users most affected by errors
+
+### Issues
+Choosing "Issues (States, Assignment, Time, etc.)" allows you to query issue properties, such as `is:unresolved` or event properties such as `error.handled:0`, and to sort by issue fields such as `First Seen`. When searching on event properties, the search will return any issue that has one or more events matching the supplied event filters. 
+Choose this dataset if you want to customize a list of issues on your dashboard. Some widget examples include:
+
+- Issues assigned to your team 
+- Most frequently occurring issues
+- The "Issues" dataset is only available in table visualization widgets and its disabled if you have other visualizations selected.
+
+## Choose Your Columns
+
+The option to set columns is only available for [table visualizations](/product/dashboards/widget-builder/#table).
+
+By default, the first column is already configured for you, but you can update it. To add more columns, which stacks events, add [functions](/product/discover-queries/query-builder/#stacking-functions) (`f(x)`) that can take in parameters. [Field and tag columns](/product/sentry-basics/search/searchable-properties/?_ga=2.209391633.2075508021.1652792062-1379607348.1646725525#event-properties) will help you view more details about the events (for example, title).
```

Comment:
Let's add at least one more example of a field or tag column in the parentheses at the end of this example.